src/Microsoft.Data.Analysis/PrimitiveDataFrameColumn.Sort.cs

```diff
@@ -13,10 +13,10 @@ namespace Microsoft.Data.Analysis
     public partial class PrimitiveDataFrameColumn<T> : DataFrameColumn
         where T : unmanaged
     {
-        public new PrimitiveDataFrameColumn<T> Sort(bool ascending = true)
+        public new PrimitiveDataFrameColumn<T> OrderBy()
```

Comment:
We also should add `OrderByDescending()` here and in `StringDataFrameColumn`. We are losing the capability to sort descending and keep the same type.